pkgs/tools/text/ripgrep/default.nix

```diff
@@ -15,6 +15,11 @@ buildRustPackage rec {
 
   depsSha256 = "0q68qyl2h6i0qsz82z840myxlnjay8p1w5z7hfyr8fqp7wgwa9cx";
 
+  preFixup = ''
+    mkdir -p "$out/man/man1"
```

Comment:
I noted that the `fixup` phase ["moves the man/, doc/ and info/ subdirectories of $out to share/"][1], and assumed that meant that `$out/man` is where the system wants me to put man-pages.

[1]: <https://nixos.org/nixpkgs/manual/#ssec-fixup-phase>